user.go

```diff
@@ -99,12 +99,29 @@ func (s *UserService) GetGroups(username string) (*[]UserGroup, *Response, error
 	return userGroups, resp, nil
 }
 
+// Get information about the current logged-in user
+//
+// JIRA API docs: https://developer.atlassian.com/cloud/jira/platform/rest/#api-api-2-myself-get
+func (s *UserService) GetSelf() (*User, *Response, error) {
+	const apiEndpoint = "rest/api/2/myself"
+	req, err := s.client.NewRequest("GET", apiEndpoint, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	var user User
+	resp, err := s.client.Do(req, &user)
+	if err != nil {
+		return nil, resp, NewJiraError(resp, err)
+	}
+	return &user, resp, nil
+}
+
 // Find searches for user info from JIRA:
 // It can find users by email, username or name
 //
 // JIRA API docs: https://docs.atlassian.com/jira/REST/cloud/#api/2/user-findUsers
 func (s *UserService) Find(property string) ([]User, *Response, error) {
-	apiEndpoint := fmt.Sprintf("/rest/api/2/user/search?username=%s", property)
+	apiEndpoint := fmt.Sprintf("/rest/api/2/user/search?username=%s&maxResults=1000", property)
```

Comment:
I don't think we should hardcode config options.  This should be part of a config struct that's passed into every function call.  Look at some of the other services that have config structs for an example.